themes/fast-track-theme/lms/static/sass/course/ccx_coach/_dashboard.scss

```diff
@@ -0,0 +1,134 @@
+@import 'create_ccx_page';
+@import 'team_management';
+@import 'enrollment';
+@import 'student_admin';
+@import 'coach_dashboard_edit_ccx';
+
+body.view-affiliate-dash {
+  background: none;
+
+  .coach.container {
+    max-width: 80%;
+    margin-bottom: 80px;
+  }
+  .window-wrap {
+    position: relative;
+    min-height: 100vh;
```

Comment:
In that case it will behave the same way as without fallback on min-height: 100% ... unfortunately, that is the reason I used vh....